packaging/testbed-server/README.md

```diff
@@ -0,0 +1,147 @@
+| D-Bus Secret Service                 | <https://github.com/docker/docker-credential-helpers/releases> | 🐧 Linux  |
+| Pass                                 | <https://github.com/docker/docker-credential-helpers/releases> | 🐧 Linux  |
+| Microsoft Windows Credential Manager | <https://github.com/docker/docker-credential-helpers/releases> | Windows  |
+| Apple macOS keychain                 | <https://github.com/docker/docker-credential-helpers/releases> | 🍎 macOS  |
+
+For the setup for each store look at <https://github.com/docker/docker-credential-helpers/blob/master/README.md>.
+
+I will detail the setup for `pass` since it's the one I'm using.
+
+##### Setup `pass` credentials store
+
+We consider you already installed [`pass`](https://www.passwordstore.org/) and initialized `pass` with a gpg key (you can take a look at <https://wiki.archlinux.org/title/Pass>).
+
+Download the artifact `docker-credential-pass-v{{ version }}.linux-{{ arch }}` at <https://github.com/docker/docker-credential-helpers/releases/latest>
+
+After you complete the download, you need to install the binary somewhere accessible in your path.
+I put mine in `~/.local/bin`, so I have the following output with the command `whereis docker-credential-pass`
+
+```shell
+$ whereis docker-credential-pass
+docker-credential-pass: /home/.../.local/bin/docker-credential-pass
+```
+
+> If your have `docker-credential-pass:` with noting after it can mean:
+>
+> - You put the binary in a folder that isn't already in the env var `PATH` (note `~/.local/bin` isn't always present by default in `PATH`).
+> - You copied the binary in a correct folder but with the wrong name.
+
+And the command `docker-credential-pass version` work
+
+```shell
+$ docker-credential-pass version
+docker-credential-pass (github.com/docker/docker-credential-helpers) v0.7.0
+```
+
+Modify the docker config file at `~/.docker/config.json`, you need to set the key `credsStore` to `pass`.
+
+```json
+{
+  "credsStore": "pass"
+}
+```
+
+Then excute `docker login`
+
+```shell
+$ docker login ghcr.io
+Username: <USERNAME>
+Password: <PAT>
+Login Succeeded
+```
+
+You can then verify, that you correct configured docker with
+
+```shell
+$ docker-credential-pass list
+{"ghcr.io": "<USERNAME>"}
+$ pass list docker-credential-helpers
+docker-credential-helpers
+└── <ghcr.io in base64>
+    └── <USERNAME>
+```
+
+### Build the docker
+
+To build the docker image, simply execute
+
+```shell
+bash packaging/testbed-server/build.sh
+```
+
+> If you are adventurous, you can replace the `bash` by `source` if you're already in a bash shell (will close if something fail, use `set +x` after).
+> That will provide access to the env var `PREFIX` & `UNIQ_TAG`.
+
+The script you give you the instruction to run the generated docker image (look for the command after `You can now test/use the docker image with:`).
+
+```shell
+docker --port 6777:6777 --rm --name=parsec-testbed-server $PREFIX/parsec-testbed-server:$UNIQ_TAG
```

Comment:
I think you forgot the `run`. 

Also, the `--port` option is not recognized by Docker, do you mean [`--publish|-p`](https://docs.docker.com/engine/reference/commandline/run/#publish) ?  

```suggestion
docker run --publish 6777:6777 --rm --name=parsec-testbed-server $PREFIX/parsec-testbed-server:$UNIQ_TAG
```

/!\ This may need to be fixed also in `build.sh`